docs/business-apps/power-automate/get-started/connect-to-other-services-in-your-flow.md

```diff
@@ -1,67 +1,63 @@
 ---
 title: Connect your flow to other services
 description: Connect your flow to 200+ services available in Power Automate, and build an integrated experience to perform actions in the services.
-ms.date: 12/12/2018
+ms.date: 3/11/2020
 ms.prod: sharepoint
 localization_priority: Priority
 ---
 
 # Connect your flow to other services
-
 Connect your flow to 200+ services available in Power Automate and build an integrated experience to perform actions in the services. This article continues building the *Projects* list flow built in the previous article *Create your first flow*.
 
 ## Connect to Microsoft Teams
-
 In this article, we will connect to Microsoft Teams and post a message to one of the channels.
 
 > **Note:** While this article focuses on Microsoft Teams, you can also connect to various other services in your flow.
 
 ### Edit the flow
-
 1. In your browser, switch to the **flow designer** tab.
 
 2. If you do not have the flow designer tab opened, to open the flow, follow these steps:
     * Browse to the **Projects** list.
-    * Select **Flow**, and then select **See your flows**.
-    * If promoted, sign in with your Microsoft 365 account.
-    * In the **My flows** tab, select **Send an email for new projects** flow.
-    * To edit the flow, select **Edit flow**.
+    * Select **Automate**, then select **Power Automate** and then expand to select **See your flows**.
+    * If prompted, sign in with your Microsoft 365 account.
+    * In the **My flows** tab, select **Send a customized email when a new SharePoint list item is added** flow.
+    * To edit the flow, in the flow details page, select **Edit** in the top command bar.
 
 3. In the flow designer, select **+ New step**.
 
-4. Select **Add an action**. The actions list opens where you can search and browse for various actions available in your flow in Power Automate.
+4. In the search box, to search for Teams-based actions, enter **Teams**.
 
-6. In the search box, to search for Teams-based actions, enter **Teams**.
+5. In the results, to filter the actions to just Teams, select the **Microsoft Teams** connector.
 
-7. In the results, to filter the actions to just Teams, select the **Microsoft Teams** connector.
+    ![Search for Microsoft Teams connector](../../../images/gs02-choose-action-microsoft-teams-connector.png)
 
-8. In the available actions list, select **Microsoft Teams - Post message**.
+6. In the available actions list, select **Microsoft Teams - Post message**.
     > To read more about the action and what it does, select the **information** icon.
```

Comment:
To read more about the action and what it does, select the **i** (information) icon.